arangod/VocBase/vocbase.cpp

```diff
@@ -1112,8 +1112,6 @@ std::shared_ptr<arangodb::LogicalDataSource> TRI_vocbase_t::lookupDataSource(
 
 /// @brief looks up a view by identifier
 std::shared_ptr<arangodb::LogicalView> TRI_vocbase_t::lookupView(TRI_voc_cid_t id) const {
-  TRI_ASSERT(!ServerState::instance()->isCoordinator());
```

Comment:
actually why are you removing these assertions ?

Reply:
because tests were failing due to code that made those calls and didn't expect to see any view instance, but instead triggered an abort
the code for collections does not have such assertions, also there could be implementations that store views in vocbase, e.g. the IResearchDBServer implementation in arangodb 3.4
so the assertion presence is not actually valid

Reply:
actually you are correct, since the assertion is 
TRI_ASSERT(!ServerState::instance()->isCoordinator());
instead of 
TRI_ASSERT(!ServerState::instance()->isClusterRole());
it is a valid assertion since the coordinator should not store anything in TRI_vocbase_t